package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-firebase-driver-testing",
-  "version": "1.0.5",
+  "version": "1.0.6",
```

Comment:
What's the release process? How does this worK?

Reply:
This was done by running `yarn version`